LEAF_Nexus/sources/Group.php

```diff
@@ -439,6 +455,9 @@ public function listGroupEmployeesDetailed($groupID, $offset = 0, $limit = -1)
     {
         // Cannot use $this->listGroupEmployees() since that query does not
         // include Employee position data
+        if(!is_numeric($groupID)) {
+            return array();
```

Comment:
Since `Exceptions` are thrown elsewhere when the input is invalid, instead of returning an empty array, it should throw `new Exception('invalid group');`